Cesium.CodeGen.Tests/CodeGenMethodTests.cs

```diff
@@ -108,6 +108,20 @@ public void IncorrectOverrideCliImport() => DoesNotCompile(@"__cli_import(""Syst
 int console_read(void);
 int console_read(void) { return 0; }", "Function console_read already defined as immutable.");
 
+    [Fact]
+    public void CanHaveTwoCliImportDeclarations() => DoTest(@"__cli_import(""System.Console::Read"")
+int console_read(void);
+__cli_import(""System.Console::Read"")
+int console_read(void);");
+
+    [Fact]
+    public void CanHaveTwoFucntionDeclarations() => DoTest(@"
+int console_read(void);
+
+int console_read(void);
+
+int console_read(void) { return 0; }");
+
```

Comment:
Let's also add a test that doesn't compile (if the function signatures of `__cli_import` clauses are different)?